org-formation/075-security-hub/security-hub.yaml

```diff
@@ -3,6 +3,10 @@ Description: "Setup AWS Security Hub"
 Resources:
   SecurityHub:
     Type: "AWS::SecurityHub::Hub"
+  SecurityHubStandard:
```

Comment:
We currently have `aws-foundational-security-best-practices/v/1.0.0` standard  enabled.  I think this change may enable `cis-aws-foundations-benchmark/v/1.4.0` and disable the security best practices one.  I think you may need to create a resource for each standard to enable both standards.

Reply:
Yes, we ca add this.